cylc/flow/task_queues/independent.py

```diff
@@ -54,7 +56,10 @@ def release(self, active: Counter[str]) -> List['TaskProxy']:
             except IndexError:
                 # deque empty
                 break
-            if itask.state.is_held:
+            if (
+                itask.state.is_held or
+                (is_paused and not itask.is_manual_submit)
+            ):
```

Comment:
We would expect this functionality to work the same across all queue implementations, so we could really do with abstracting it into the base class rather than relying on each queue to implement it separately.

Idea (for future work), write some abstract tests that should hold for all queue implementations.

Reply:
Good idea. Opened an Issue for that https://github.com/cylc/cylc-flow/issues/6483